lib/omnibus/health_check.rb

```diff
@@ -552,11 +553,11 @@ def check_for_bad_macos_library(bad_libs, current_library, name, linked)
           #         name /opt/datadog-agent/embedded/lib (offset 12)
           yield_shellout_results("otool -l #{current_library} | grep LC_RPATH -A2 | grep path | awk '{ print $2 }'") do |rpath|
             # The rpath variable contains a \n (\r\n on Windows), so we remove it when including it in the complete path
-            possible_paths << linked.sub("@rpath", rpath.chop)
+            # If @rpath contains @loader_path we're replacing it with it's actual loader_path
```

Comment:
Just a suggestion for more concise wording, feel free to ignore.
```suggestion
            # We also resolve possible `@loader_path` references
```